docker-compose.yml

```
@@ -4,19 +4,23 @@ services:
   reverse-proxy:
     image: traefik:v2.3
     environment:
+      - CONFIG_LOCATION=/dynamic/${TRANSFER_PROTOCOL}/traefik.toml
       - ACCESS_CONTROL_PROTOCOL
       - ACCESS_CONTROL_HOST
       - SCA_API_PROTOCOL
       - SCA_API_HOST
       - HTTPS_PROXY
       - NO_PROXY
+    command: --configFile $${CONFIG_LOCATION}
     ports:
-      - "${EXTERNAL_PORT}:80"
+      - "${EXTERNAL_PORT}:${TRAEFIK_PORT}"
       - "${TRAEFIK_DASHBOARD_PORT}:8080"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - ./volumes/traefik/traefik.toml:/etc/traefik/traefik.toml
-      - ./volumes/traefik/traefik-dynamic.yml:/dynamic/traefik-dynamic.yml
+      - ./volumes/traefik/http:/dynamic/http
+      - ./volumes/traefik/https:/dynamic/https
+      - ./volumes/traefik/ssl/cert.crt:/etc/traefik/ssl/cert.crt
+      - ./volumes/traefik/ssl/key.key:/etc/traefik/ssl/key.key
```

```suggestion
      - ./volumes/traefik:/etc/traefik
```

Can you make that work?

---

no... balagan it overrides files of traefik